src/constants.cpp

```diff
@@ -94,6 +94,7 @@ QString Constants::apiUrl(ApiEndpoint endpoint) {
 #endif
 #ifdef MZ_IOS
       {ApiEndpoint::PurchasesIOS, "/api/v1/vpn/purchases/ios"},
+      {ApiEndpoint::PurchasesIOSv2, "/api/v2/vpn/purchases/ios"},
```

Comment:
Is there a corresponding PR to the guardian website that I can compare against?

Reply:
It was added here: https://github.com/mozilla-services/guardian-website/pull/1541/